pkg/fleet/installer/installer.go

```diff
@@ -301,32 +306,40 @@ func (i *installerImpl) stopExperiment(ctx context.Context, pkg string) error {
 	}
 }
 
-func (i *installerImpl) setupPackage(ctx context.Context, pkg string) error {
+func (i *installerImpl) promoteExperiment(ctx context.Context, pkg string) error {
+	switch pkg {
+	case packageDatadogAgent:
+		return service.PromoteAgentExperiment(ctx)
+	/*case packageAPMInjector:
```

Comment:
You are breaking the experiment for the installer here, you need to add an entry for each package that was supported in `stopExperiment`. You can continue to call their package-specific `stopExperiment`, no need to create a new promote for each of them

Reply:
There's a small additional issue, I'll let you merge the proposed change